tensorflow/lite/kernels/fully_connected.cc

```diff
@@ -1329,6 +1337,11 @@ TfLiteStatus EvalQuantized(TfLiteContext* context, TfLiteNode* node,
                       GetTensorData<int32_t>(bias), GetTensorShape(output),
                       GetTensorData<int16_t>(output),
                       CpuBackendContext::GetFromContext(context));
+          } else {
+            is_per_channel ? FullyConnectedPerChannelInt16<kernel_type>(
+                                 data, input, filter, bias, output)
+                           : FullyConnectedInt16<kernel_type>(
+                                 data, input, filter, bias, output);
           }
         } else if (kernel_type == kReference) {
```

Comment:
This has broken an internal test. Could you please explain the change in logic here?